### PR TITLE
CD & Inventory Test Update: Integrate cfgplugins & Address Bugs

### PR DESCRIPTION
**Pull request by yiwenhu-g**
- Integrate cfgplugins
- Remove outdated deviations
- Address Bugs

### CODE REVIEW

File: feature/platform/transceiver/tests/zr_cd_test/metadata.textproto [hunk 1]
```diff
@@ -12,5 +12,6 @@ platform_exceptions: {
     interface_enabled: true
     default_network_instance: "default"
     missing_port_to_optical_channel_component_mapping: true
+    missing_zr_optical_channel_tunable_parameters_telemetry: true
```
Comment (ahsaanyousaf) [inline feature/platform/transceiver/tests/zr_cd_test/metadata.textproto:15]: Do we still need these exceptions for this?

Comment (yiwenhu-g) [reply feature/platform/transceiver/tests/zr_cd_test/metadata.textproto:15]: Changed to false

**Comment by yiwenhu-g** [comment]
Internal tests PASS
```
sponge2/bbd54639-c8a5-4c58-88b1-8f1e7d5b8eba
sponge2/0ae95a7e-ae81-4c37-bb5e-20fed5ca9f73
```

**Comment by yiwenhu-g** [comment]
Internal tests PASS
- Inventory Test: sponge2/bbd54639-c8a5-4c58-88b1-8f1e7d5b8eba
- CD Test: sponge2/a281f9bb-ce69-473e-8f07-a6d9453ef7b4

**Review by ahsaanyousaf — APPROVED** [review]

**Comment by dplore** [comment]
Thanks for contributing to cfgplugins

**Pull request merged**